recipes-bsp/trusted-firmware-a/trusted-firmware-a_git.bb

```diff
@@ -1,8 +1,10 @@
 require recipes-bsp/trusted-firmware-a/trusted-firmware-a.inc
 
 # Current master branch
-SRCREV_tfa = "09a1cc2a0bd066702daa269bf35de9c5743ccc93"
-SRCBRANCH = "master"
+SRC_URI_TRUSTED_FIRMWARE_A = "git://github.com/xen-troops/arm-trusted-firmware.git;protocol=https"
+
+SRCREV_tfa = "${AUTOREV}"
```

Comment:
Please do not use `AUTOREV` as it causes problems during releases

Reply:
My comment here is the same as in PR#https://github.com/xen-troops/meta-xt-rpi5/pull/34. Once at least some level of active development is done it can be fixed.